<!-- Safety launch file -->

<launch>
	<group ns="RC_Safety">
		<node pkg="rc_safety" name="Safety" type="rc_safety">
			<param name="safety_pub" value="/rcSafety/safetyStatus"/>
		</node>
	</group>
</launch>
